release_tester/arangodb/starter/deployments/activefailover.py

```diff
@@ -395,3 +396,19 @@ def set_selenium_instances(self):
             [x for x in self.leader.all_instances if x.instance_type == InstanceType.RESILIENT_SINGLE][0],
             self.new_cfg,
         )
+
+    def get_agency_leader_starter(self):
```

Comment:
don't we already have a very similar function in runner.py? I think we should have DRY here, and implement this as part of the runner class.